src/test/util/net.h

```diff
@@ -133,110 +133,77 @@ constexpr auto ALL_NETWORKS = std::array{
     Network::NET_INTERNAL,
 };
 
+std::vector<NodeEvictionCandidate> GetRandomNodeEvictionCandidates(int n_candidates, FastRandomContext& random_context);
+
 /**
- * A mocked Sock alternative that returns a statically contained data upon read and succeeds
- * and ignores all writes. The data to be returned is given to the constructor and when it is
- * exhausted an EOF is returned by further reads.
+ * A mocked Sock alternative that succeeds on all operations.
+ * Returns infinite amount of 0x0 bytes on reads.
  */
-class StaticContentsSock : public Sock
+class ZeroSock : public Sock
 {
 public:
-    explicit StaticContentsSock(const std::string& contents)
-        : Sock{INVALID_SOCKET},
```

Comment:
No, the new `ZeroSock` constructor's body is now in `src/test/util/net.cpp` and it does

```cpp
ZeroSock::ZeroSock() : Sock{g_mocked_sock_fd++} {}
```

Because if all of the `ZeroSock` instances have `m_socket == INVALID_SOCKET`, then they would compare equal by `EqualSharedPtrSock`.